examples/virtual_staining/vcp_tutorials/quick_start.py

```diff
@@ -0,0 +1,258 @@
+
+## Example Dataset
+
+The example dataset contains quantitative phase and paired nuclei and plasma membrane fluorescence images of A549 cells.
+It is stored in OME-Zarr format and can be downloaded from
+[here](https://public.czbiohub.org/comp.micro/viscy/VS_datasets/VSCyto2D/test/a549_hoechst_cellmask_test.zarr).
+It has pre-computed statistics for normalization, generated using the `viscy preprocess` CLI.
+
+Refer to our [preprint](https://doi.org/10.1101/2024.05.31.596901) for more details
+about how the dataset and model were generated.
+"""
+
+# %% [markdown]
+"""
+# Setup
+
+The commands below will install the required packages and download the example dataset and model checkpoint.
+It may take a few minutes to download all the files.
+Assuming a Unix-like shell.
+"""
+
+# %%
+# Install VisCy with the optional dependencies for this example
+# See the [repository](https://github.com/mehta-lab/VisCy) for more details
+# !pip install "viscy[metrics,visual]==0.3.0rc2"
+
+# %%
+# Validate installation
+# !viscy --help
+
+# %%
+# Download the example dataset
+# !wget -m -np -nH --cut-dirs=5 -R "index.html*" "https://public.czbiohub.org/comp.micro/viscy/VS_datasets/VSCyto2D/test/a549_hoechst_cellmask_test.zarr/"
+# Download the model checkpoint
+# !wget https://public.czbiohub.org/comp.micro/viscy/VS_models/VSCyto2D/VSCyto2D/epoch=399-step=23200.ckpt
+
+# %% [markdown]
+"""
+# Run Model Inference
+
+The following code will run inference on a single field of view (FOV) of the example dataset.
+This can also be achieved by using the VisCy CLI.
+"""
+
+# %%
+from pathlib import Path
+
+from iohub import open_ome_zarr
+from torchview import draw_graph
+
+from viscy.data.hcs import HCSDataModule
+from viscy.trainer import VisCyTrainer
+from viscy.transforms import NormalizeSampled
+from viscy.translation.engine import FcmaeUNet
+from viscy.translation.predict_writer import HCSPredictionWriter
+
+# %%
+# TODO: Set download paths, by default the working directory is used
+root_dir = Path()
+# TODO: modify the path to the downloaded dataset
+input_data_path = root_dir / "a549_hoechst_cellmask_test.zarr"
+# TODO: modify the path to the downloaded checkpoint
+model_ckpt_path = root_dir / "epoch=399-step=23200.ckpt"
+# TODO: modify the path
+# Zarr store to save the predictions
+output_path = root_dir / "a549_prediction.zarr"
+# TODO: Choose an FOV
+fov = "0/0/0"
+
+input_data_path = input_data_path / fov
+
+# %%
+# Create the VSCyto2D network
+
+# Reduce the batch size if encountering out-of-memory errors
+BATCH_SIZE = 8
+# NOTE: Set the number of workers to 0 for Windows and macOS
+# since multiprocessing only works with a
+# `if __name__ == '__main__':` guard.
+# On Linux, set it to the number of CPU cores to maximize performance.
+NUM_WORKERS = 0
+phase_channel_name = "Phase3D"
+
+# %%
+# See API documentation for more details. For example:
+# ?HCSDataModule
+
+# %%
+# Setup the data module.
+data_module = HCSDataModule(
```

Comment:
We can add links to PyTorch Lightning documentation here to give user some context.

Reply:
We can also write a utility method for running inference, which takes input_path, output_path, config, and checkpoint as inputs.

Reply:
Added more comments and help message blocks.